src/main/java/org/symphonyoss/integration/jira/services/SearchAssignableUsersService.java

```diff
@@ -0,0 +1,55 @@
+package org.symphonyoss.integration.jira.services;
+
+import static org.symphonyoss.integration.jira.properties.ServiceProperties.APPLICATION_KEY_ERROR;
+
+import com.google.api.client.http.HttpMethods;
+import com.google.api.client.http.HttpResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.integration.authorization.oauth.v1.OAuth1Exception;
+import org.symphonyoss.integration.authorization.oauth.v1.OAuth1HttpRequestException;
+import org.symphonyoss.integration.authorization.oauth.v1.OAuth1Provider;
+import org.symphonyoss.integration.jira.exception.JiraAuthorizationException;
+import org.symphonyoss.integration.logging.LogMessageSource;
+import org.symphonyoss.integration.model.ErrorResponse;
+
+import java.io.IOException;
+import java.net.URL;
+
+/**
+ * Created by alexandre-silva-daitan on 15/08/17.
+ */
+@Component
+public class SearchAssignableUsersService {
+
+  @Autowired
+  private LogMessageSource logMessage;
+
+  public ResponseEntity searchAssingablesUsers(String accessToken, OAuth1Provider provider,
```

Comment:
Missing javadoc.

You don't need to pass component as a parameter, this a constant. You could define it as a constant field in a separate class that holds the constant fields.